src/styles/components/_anchor.scss

```diff
@@ -41,15 +35,10 @@ a {
         @include di("download");
       }
 
-      &:hover {
-        &::after {
-          @include di-variant("download-scampi");
-        }
-      }
-
+      &:hover,
       &:focus {
         &::after {
-          @include di-variant("download-wit");
+          @include di-variant("download-scampi");
```

Comment:
Er is nu geen rekening gehouden met `:focus` op `:visited`: 

![image](https://user-images.githubusercontent.com/9696261/67671359-1711a400-f976-11e9-8ab0-1fba2e37c52a.png)

Prima als we dit een andere keer oplossen, maar we maken nota bene zelf `:visited` styling maar vangen deze situatie dan niet af

Reply:
Overigens ziet het er in versie 8.3.0 zo uit:

![image](https://user-images.githubusercontent.com/9696261/67671825-eda54800-f976-11e9-8120-b735879549c3.png)